### PR TITLE
feat: add SUI API

### PR DESCRIPTION
**Pull request by ladovina**

### CODE REVIEW

File: blockapi/v2/api/sui.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                decimals=response_coin.get('decimals'),
+                blockchain=Blockchain.SUI,
+                address=response_coin.get('type'),
+                info=None,  # TODO
+            )
+
+            yield BalanceItem.from_api(
+                balance_raw=response_coin.get('objectCount', 0),
+                coin=self.coin,
+                raw=coin,
```
Comment (galvanizze) [inline blockapi/v2/api/sui.py:44]:
```suggestion
                raw=response_coin,
```

File: blockapi/v2/api/sui.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    def get_balance(self, address: str) -> List[BalanceItem]:
+        balances = list(self._yield_available_balance(address))
+        return balances
+
+    def _yield_available_balance(self, address: str) -> Iterable[BalanceItem]:
+        response = self._get('get_balances', address=address)
+
+        for response_coin in response.get('coins', list()):
+            coin = Coin(
```
Comment (galvanizze) [inline blockapi/v2/api/sui.py:32]: nitpick: better to use class method `Coin.from_api`

File: blockapi/v2/api/sui.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                symbol=response_coin.get('symbol'),
+                name=response_coin.get('name'),
+                decimals=response_coin.get('decimals'),
+                blockchain=Blockchain.SUI,
+                address=response_coin.get('type'),
```
Comment (galvanizze) [inline blockapi/v2/api/sui.py:37]: is address really stored in `type` field? Sorry I didn't test it

File: blockapi/v2/api/sui.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                info=None,  # TODO
```
Comment (galvanizze) [inline blockapi/v2/api/sui.py:38]: 🙂

File: blockapi/v2/api/sui.py (hunk 1 above)
Comment (ladovina) [reply blockapi/v2/api/sui.py]: fixed

File: blockapi/v2/api/sui.py (hunk 2 above)
Comment (ladovina) [reply blockapi/v2/api/sui.py:32]: changed

File: blockapi/v2/api/sui.py (hunk 4 above)
Comment (ladovina) [reply blockapi/v2/api/sui.py]: removed `info` and `todo`

**Review by galvanizze — APPROVED** [review]

**Pull request merged**